src/GravityFormsExtensions.php

```diff
@@ -97,6 +98,10 @@ private function hooks(): void
         add_filter('gform_field_css_class', [ $this, 'p4_gf_custom_field_class' ], 10, 3);
         add_filter('gform_form_args', [ $this, 'p4_gf_enforce_ajax' ], 10, 3);
         add_action('gform_after_save_form', [ $this, 'p4_gf_clear_page_caches' ], 10, 2);
+
+        // Suppress the redirect in forms to use custom redirect handling.
+        add_filter('gform_suppress_confirmation_redirect', '__return_true');
+        add_filter('gform_confirmation', [ $this, 'p4_gf_custom_confirmation_redirect' ], 10, 2);
```

Comment:
[We already use this hook](https://github.com/greenpeace/planet4-master-theme/blob/main/src/GravityFormsExtensions.php#L96), so maybe we could combine the functionalities into one function?

Reply:
Yes, we can but keeping whole logic in one function will make a one function too big, using a separate function will make functionality more moduler. also we have option to priorities(3rd arg in `add_filter()` function) which filter function execute first so we can decide execution flow as well, if we need.